### ISSUE TITLE
storage: benignError printed as regular error

### ISSUE THREAD

**Issue by andreimatei** [cockroachdb/cockroach#34101]
This ostensibly `benignError` is printed to the log as a scary error:
https://github.com/cockroachdb/cockroach/blob/9701846a720b323409d24fe537266a7394522551/pkg/storage/replicate_queue.go#L383
```
E190117 02:11:45.902661 24634 storage/queue.go:793  [n4,replicate,s4,r22640/3:/Table/67/7/"10061{049…-269…}] no removable replicas from range that needs a removal: [raft progress unknown]
```

We should do something, but I'm not sure what. Does it need to be an error at all?

**Comment by knz** [comment]
This has been fixed in #31728.

The tree that you Andrei link to comes from a commit (2558dccbcda09c0eae28f816a05adb8620155175)  that pre-dates #31728.

You probably found that error in a cluster running a version of cockroachdb older than that.

Closing.

**Comment by tbg** [comment]
Thanks for investigating, Raphael!
